media/js/firefox/australis/tour.js

```diff
@@ -3,6 +3,12 @@
 
     var firstTime = 'True';
 
+    // GA tracking for Firstrun tests
```

Comment:
These should probably be wrapped in a version check for 35, as this file is not unique to these tests? That said, could these vars not be housed elsewhere?